FinniversKit/Sources/Components/JobKeyInfo/Subviews/JobKeyInfoView+Separator.swift

```diff
@@ -0,0 +1,23 @@
+import UIKit
+
+extension JobKeyInfoView {
+    class Separator: UIView {
+
+        // MARK: - Init
+
+        override init(frame: CGRect) {
+            super.init(frame: frame)
+            setup()
+        }
+
+        required init?(coder: NSCoder) { fatalError() }
```

Comment:
Just to follow same style, but not necessary: "fatalError("init(coder:) has not been implemented")"